Figure 2 endemic

geethaj1 opened the pull request:
Previously the UVC timestep was set to turn on at time = 0. Setting timestep = 365*18 would allow us to use years 15-17 as a baseline and compare it to years 18=20 when UV-C is turned on. This would also match the timestep used in the blueprint report

GeethaJey committed:
Comment edits for figure layour

GeethaJey committed:
updated comments

Commit:
Added scripts and architecture to run simlations from saved parameter lists on the cluster via hipercow

Commit:
Added description to figure_2_simulations_hipercow.R and tidied formatting in hipercow scripts

Commit:
Amended figure_2_parameter_list.R to correcty assign simulation identifiers

athowes committed:
Use air formatter

tbreweric committed:
Update figures/figure_2/figure_2_simulations_hipercow.R

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

tbreweric committed:
Update figures/figure_2/figure_2_simulations_hipercow.R

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Commit:
Moved run_simulations_hipercow function to new run.R script in R/ and updated figure_2_simulations_hipercow.R to accommodate change

tbreweric committed:
Apply suggestions from code review

Batch commit of formatting suggestions for run_simulation_hipercow() function in run.R

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

athowes committed:
Format

GeethaJey committed:
bringing up to speed with adam's annotations

Merge branch 'figure_2_endemic' of github.com:petal-code/helios-paper into figure_2_endemic

# Conflicts:
#	figures/figure_2/figure_2_parameter_list.R

cwhittaker1000 committed:
Merge branch 'figure_2_endemic' into figure_2_simulations

cwhittaker1000 committed:
Merge pull request #29 from petal-code/figure_2_simulations

merge figure 2 simulations branch into figure 2 endemic

GeethaJey committed:
Updates to code per conversation with charlie and tom

GeethaJey committed:
Figure 2 Plot

GeethaJey committed:
update gitignore

geethaj1 committed:
Figure 2 draft plots and update to parameter list generation

geethaj1 committed:
Updated figure 2 simulation parameter list with proper UVC timestep

geethaj1 committed:
updates to figure 2 plot

geethaj1 committed:
Edits to figure 3 plots based on feedback

geethaj1 committed:
Adjustment from total population to prevalence for active infections

geethaj1 committed:
Adjusted error bars to be min/max rather than IQR, adjusted colours, adjusted prevalence to be reported as percentage rather than proportion

geethaj1 committed:
adjust y axis

geethaj1 committed:
update windows and size of workplaces

geethaj1 committed:
Delete figures/figure_2/figure_2_simulations.R

geethaj1 committed:
Delete figures/figure_2/figure2_parameter_combinations.rds

geethaj1 committed:
Delete figures/figure_2/figure2_parameter_list.rds

geethaj1 committed:
edits to population size and uvc turn on time

geethaj1 committed:
edit

geethaj1 committed:
local updates before rebase

geethaj1 committed:
widen heat map

geethaj1 committed:
Delete figures/figure_3 directory

Removing figure 3 files from the figure 2 branch as we there is a separate branch for figure 3 parameter list generation + plots

geethaj1 committed:
Rename final_figure_2_plots.R to figures/figure_2/final_figure_2_plots.R

Cleaning up folders

geethaj1 committed:
Delete Rplot.png

geethaj1 committed:
Delete Rplot01.png

geethaj1 committed:
Delete Rplot02.png

geethaj1 committed:
Delete figure3_option2.png

geethaj1 committed:
Delete figure_3_plots.R

geethaj1 committed:
Create placeholder.txt

geethaj1 committed:
Rename figure_2_parameter_list.rds to figures/figure_2/old/figure_2_parameter_list.rds

geethaj1 committed:
Delete figure_3_parameter_list.rds

geethaj1 committed:
Delete figure_3_parameter_combinations.rds

geethaj1 committed:
Delete figure_panels.png

geethaj1 committed:
Rename figures/figure_2/Figure_2_plots.R to figures/figure_2/old/Figure_2_plots.R

geethaj1 committed:
Delete .gitignore

geethaj1 committed:
Delete R/run.R

geethaj1 committed:
Delete figures/figure_2/figure_2_parameter_combinations.rds

geethaj1 committed:
Delete figures/figure_2/.gitignore

geethaj1 committed:
Delete random.png

geethaj1 committed:
Delete figures/figure_2/old/placeholder.txt

athowes commented on .gitignore:
Could you not delete this please? The `.gitignore` file prevents us commiting certain types of file to the repo

athowes commented on .RDataTmp:
I don't think this should be commited here

geethaj1 replied:
should I delete the file? or move it to a different location? I'm not sure what this file is

athowes replied:
Delete. It's some data from your R workspace

athowes commented on parameter_combinations.rds:
This should either be in the `figure_2/` folder or deleted

athowes commented on figures/figure_2/final_figure_2_plots.R:
One of the benefits of using Git is that we don't need to call files "final_" "final_really_this_time_" etc. We can just make changes to the file and then the edits are all in the commit history. So I'd recommend deleting the "old" folder (you can always find them in the commit history) and renaming this to get rid of "final" notions

athowes commented on figures/figure_2/final_figure_2_plots.R:
```diff
@@ -0,0 +1,408 @@
+library(tidyr)
+library(ggplot2)
+library(viridis)
+library(cowplot)
```
Use `packages.R` here?

athowes commented on figures/figure_2/final_figure_2_plots.R:
```diff
@@ -0,0 +1,408 @@
+library(cowplot)
+
+
+dir_sim <- "/Users/geethaj/Downloads/figure_2_simulations"
```
I'm not `geethaj` so this directory won't work for me. Could you make it so that anyone can reproduce?

athowes replied:
[Change folder to be generic](https://github.com/petal-code/helios-paper/pull/39/commits/5647ddd516a45b794021aa56241d178262b05aa2)

athowes reviewed (changes requested):
Could you rebase this branch so that it doesn't have conflicts with main? Would help for the review. Left some other comments.

athowes commented on figures/figure_2/old/Figure_2_plots.R:
Could you run the air formatter on these files to make them consistent / easy to read? Instructions here: https://posit-dev.github.io/air/

geethaj1 committed:
Delete figures/figure_2/old directory

geethaj1 committed:
Rename final_figure_2_plots.R to figure_2_plots.R

geethaj1 committed:
Create file

geethaj1 committed:
Add files via upload

geethaj1 committed:
Delete figures/figure_2/figure_2_simulations directory

attempted to manually add rds files and found not all uploaded. will upload files through terminal

athowes committed:
Change folder to be generic

athowes committed:
Format

geethaj1 committed:
updated plots based on comments

geethaj1 committed:
save local gitignore before switching

geethaj1 committed:
edits to figure based on feedback

geethaj1 committed:
moving to new branch

geethaj1 commented:
closing as being handled in a different PR
